docs: updates from qa (#360)
phact/gendb: docs/docs/core-components/knowledge.mdx

The ingestion process may take some time, depending on the size of your documents.
4. When ingestion is complete, your documents are available in the Knowledge screen.

If ingestion fails, click on the **Status** to view the logged error.

## Monitor ingestion tasks

When you upload files, process folders, or sync documents, OpenRAG processes them as background tasks.
The <Icon name="Bell" aria-hidden="true"/> **Tasks** icon displays a notification indicator when there are active tasks running.
To open the Tasks menu, click the <Icon name="Bell" aria-hidden="true"/> **Tasks**.

**Active Tasks** displays tasks that are currently processing.

> A pending task is queued and waiting to start, a running task is actively processing files, and a processing task is performing ingestion operations. Each active task displays the task ID, start time and duration, and progress showing the number of files processed out of the total files. 

Reviewer: ```suggestion
A **Pending** task is queued and waiting to start, a **Running** task is actively processing files, and a **Processing** task is performing ingestion operations. For each active task, you can find the task ID, start time, duration, the number of files processed so far, and the total files.
```
